core/sql/optimizer/BindRelExpr.cpp

```diff
@@ -717,7 +717,7 @@ static ItemExpr *intersectColumns(const RETDesc &leftTable,
     ItemExpr *leftExpr  = leftTable.getValueId(i).getItemExpr();
     ItemExpr *rightExpr = rightTable.getValueId(i).getItemExpr();
     BiRelat *compare = new (bindWA->wHeap())
-      BiRelat(ITM_EQUAL, leftExpr, rightExpr);
+      BiRelat(ITM_EQUAL, leftExpr, rightExpr, TRUE);
```

Comment:
EXPLAIN:

>> explain select eno, ename, dno from t021
 intersect
+>+>select eno, ename, dno from t022;

------------------------------------------------------------------ PLAN SUMMARY
MODULE_NAME .............. DYNAMICALLY COMPILED
STATEMENT_NAME ........... NOT NAMED
PLAN_ID .................. 212336560049570734
ROWS_OUT ................. 2
EST_TOTAL_COST ........... 0.02
STATEMENT ................ select eno, ename, dno
                           from t021 intersect select eno, ename,
                             dno from t022;


------------------------------------------------------------------ NODE LISTING
ROOT ======================================  SEQ_NO 5        ONLY CHILD 4
REQUESTS_IN .............. 1
ROWS_OUT ................. 2
EST_OPER_COST ............ 0
EST_TOTAL_COST ........... 0.02
DESCRIPTION
  max_card_est ........... 2
  fragment_id ............ 0
  parent_frag ............ (none)
  fragment_type .......... master
  statement_index ........ 0
  affinity_value ......... 0
  max_max_cardinality    100
  total_overflow_size .... 0.00 KB
  xn_access_mode ......... read_only
  xn_autoabort_interval    0
  auto_query_retry ....... enabled
  plan_version ....... 2,600
  embedded_arkcmp ........ used
  IS_SQLCI ............... ON
  LDAP_USERNAME
  MODE_SPECIAL_4 ......... ON
  GENERATE_EXPLAIN ....... ON
  ObjectUIDs ............. 9084990775623580461, 9084990775623580331
  select_list ............ TRAFODION.SEABASE.T022.ENO,
                             TRAFODION.SEABASE.T022.ENAME,
                             TRAFODION.SEABASE.T022.DNO


NESTED_JOIN ===============================  SEQ_NO 4        CHILDREN 2, 3
REQUESTS_IN .............. 1
ROWS_OUT ................. 2
EST_OPER_COST ............ 0.01
EST_TOTAL_COST ........... 0.02
DESCRIPTION
  max_card_est ........... 2
  fragment_id ............ 0
  parent_frag ............ (none)
  fragment_type .......... master
  join_type .............. inner
  join_method ............ nested


TRAFODION_VSBB_SCAN =======================  SEQ_NO 3        NO CHILDREN
TABLE_NAME ............... T021
REQUESTS_IN .............. 1
ROWS_OUT ................. 1
EST_OPER_COST ............ 0.02
EST_TOTAL_COST ........... 0.02
DESCRIPTION
  max_card_est ........... 2
  fragment_id ............ 0
  parent_frag ............ (none)
  fragment_type .......... master
  scan_type .............. subset scan of table TRAFODION.SEABASE.T021
  object_type ............ Trafodion
  cache_size ........... 100
  probes ................. 2
  successful_probes ...... 2
  unique_probes .......... 2
  rows_accessed .......... 2
  column_retrieved ....... #1:1,#1:2,#1:3
  key_columns ............ ENO
  executor_predicates .... (ENAME = TRAFODION.SEABASE.T022.ENAME) and (DNO =
                             TRAFODION.SEABASE.T022.DNO)
  begin_key .............. (ENO = TRAFODION.SEABASE.T022.ENO)
  end_key ................ (ENO = TRAFODION.SEABASE.T022.ENO)


HASH_GROUPBY ==============================  SEQ_NO 2        ONLY CHILD 1
REQUESTS_IN .............. 1
ROWS_OUT ................. 2
EST_OPER_COST ............ 0.01
EST_TOTAL_COST ........... 0.01
DESCRIPTION
  memory_quota ........... 1200 MB
  max_card_est ........... 8
  fragment_id ............ 0
  parent_frag ............ (none)
  fragment_type .......... master
  grouping_columns ....... TRAFODION.SEABASE.T022.ENO,
                             TRAFODION.SEABASE.T022.ENAME,
                             TRAFODION.SEABASE.T022.DNO


TRAFODION_SCAN ============================  SEQ_NO 1        NO CHILDREN
TABLE_NAME ............... T022
REQUESTS_IN .............. 1
ROWS_OUT ................ 25
EST_OPER_COST ............ 0.01
EST_TOTAL_COST ........... 0.01
DESCRIPTION
  max_card_est ......... 100
  fragment_id ............ 0
  parent_frag ............ (none)
  fragment_type .......... master
  scan_type .............. subset scan of table TRAFODION.SEABASE.T022
  object_type ............ Trafodion
  columns ................ all
  begin_keys(incl)
  end_keys(incl)
  cache_size ........... 100
  probes ................. 1
  rows_accessed ........ 100
  column_retrieved ....... #1:1,#1:2,#1:3,#1:4
  key_columns ............ SYSKEY

--- SQL operation complete.


SHOWPLAN ===================================================

>>showplan select eno, ename, dno from t021
 intersect
+>+>select eno, ename, dno from t022;
-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x43e93e0 (1)
Preds:
Succs:
NULL_VIOLATION_MBIN16U  = [4,0](MBIN16U), 21 4 0
MOVE_MBIN32U_MBIN32U [2,0](MBIN32U) = [4,4](MBIN32U), 202 2 0 4 4
OPDATA_MPTR32_IBIN32S [2,0](MPTR32) =  0 2 0 10
CLAUSE_EVAL  =  108 1448768272 32524 0
OPDATA_MPTR32_IBIN32S [4,8](MPTR32) =  0 4 8 0
OPDATA_MPTR32_IBIN32S [4,10](MPTR32) =  0 4 10 10
CLAUSE_EVAL  =  108 1448768824 32524 0
OPDATA_MPTR32_IBIN32S [4,30](MPTR32) =  0 4 30 0
OPDATA_MPTR32_IBIN32S [4,32](MPTR32) =  0 4 32 10
CLAUSE_EVAL  =  108 1448769384 32524 0
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x2d845c0 (1)
Preds:
Succs: 2 3
NNB_MATTR3_IBIN32S  = [4,8,0](MPTR32), 306 4 8 0 10

BLOCK: 0x4a7bc70 (2)
Preds: 1
Succs: 4
MOVE_MBIN32S_IBIN32S [2,8](MBIN32S) =  3 2 8 666654765
BRANCH  =  95 8

BLOCK: 0x4a7cae0 (3)
Preds: 1
Succs: 4
HASH_MBIN32U_MBIN32_IBIN32S_IBIN32S [2,8](MBIN32U) = [4,24](MBIN32S), 170 2 8 4 24 0 4

BLOCK: 0x43e9c50 (4)
Preds: 3 2
Succs: 5 6
NNB_MATTR3_IBIN32S  = [4,8,1](MPTR32), 306 4 8 1 10

BLOCK: 0x43eaaa0 (5)
Preds: 4
Succs: 7
MOVE_MBIN32S_IBIN32S [2,12](MBIN32S) =  3 2 12 666654765
BRANCH  =  95 8

BLOCK: 0x3543970 (6)
Preds: 4
Succs: 7
HASH_MBIN32U_MPTR32_IBIN32S_IBIN32S [2,12](MBIN32U) = [4,32](MPTR32), 94 2 12 4 32 0 20

BLOCK: 0x3544700 (7)
Preds: 6 5
Succs: 8 9
HASHCOMB_MBIN32U_MBIN32U_MBIN32U [2,4](MBIN32U) = [2,8](MBIN32U),[2,12](MBIN32U), 156 2 4 2 8 2 12
NNB_MATTR3_IBIN32S  = [4,8,2](MPTR32), 306 4 8 2 12

BLOCK: 0x371a980 (8)
Preds: 7
Succs:
HASHCOMB_MBIN32U_MBIN32U_MBIN32U [5,0](MBIN32U) = [2,4](MBIN32U),[1,8](MBIN32U), 156 5 0 2 4 1 8
RETURN  =  264

BLOCK: 0x371b7b0 (9)
Preds: 7
Succs:
HASH_MBIN32U_MBIN32_IBIN32S_IBIN32S [2,16](MBIN32U) = [4,28](MBIN32S), 170 2 16 4 28 0 4
HASHCOMB_MBIN32U_MBIN32U_MBIN32U [5,0](MBIN32U) = [2,4](MBIN32U),[2,16](MBIN32U), 156 5 0 2 4 2 16
RETURN  =  264

BLOCK: 0x51a5280 (10)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x493a4b0 (1)
Preds:
Succs: 2 3
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [5,0](MBIN16S) = [4,8,0](MPTR32), 248 5 0 4 8 134219268 -1 0 14

BLOCK: 0x493b2a0 (2)
Preds: 1
Succs: 4
MOVE_MBIN32S_IBIN32S [5,4](MBIN32S) =  3 5 4 0
BRANCH  =  95 6

BLOCK: 0x5c67910 (3)
Preds: 1
Succs: 4
MOVE_MBIN32U_MBIN32U [5,4](MBIN32U) = [4,24](MBIN32U), 202 5 4 4 24

BLOCK: 0x5c686a0 (4)
Preds: 3 2
Succs: 5 6
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [5,8](MBIN16S) = [4,8,1](MPTR32), 248 5 8 4 8 134219268 -1 1 15

BLOCK: 0x5c69510 (5)
Preds: 4
Succs: 7
FILL_MEM_BYTES [5,10](MPTR32)[5,12](MPTR32)[5,14](MPTR32) =  115 5 10 20 0
BRANCH  =  95 7

BLOCK: 0x5c6a380 (6)
Preds: 4
Succs: 7
MOVE_MBIN8_MBIN8_IBIN32S [5,10](MPTR32) = [4,32](MPTR32), 4 5 10 4 32 20

BLOCK: 0x2984d30 (7)
Preds: 6 5
Succs: 8 9
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [5,30](MBIN16S) = [4,8,2](MPTR32), 248 5 30 4 8 134219268 -1 2 13

BLOCK: 0x2985ba0 (8)
Preds: 7
Succs:
MOVE_MBIN32S_IBIN32S [5,32](MBIN32S) =  3 5 32 0
RETURN  =  264

BLOCK: 0x2986a10 (9)
Preds: 7
Succs:
MOVE_MBIN32U_MBIN32U [5,32](MBIN32U) = [4,28](MBIN32U), 202 5 32 4 28
RETURN  =  264

BLOCK: 0x2d87830 (10)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x3717880 (1)
Preds:
Succs:
MOVE_MBIN32U_MBIN32U [5,0](MBIN32U) = [4,0](MBIN32U), 202 5 0 4 0
MOVE_MBIN32U_MBIN32U [5,4](MBIN32U) = [4,4](MBIN32U), 202 5 4 4 4
MOVE_MBIN16U_MBIN16U [5,8](MBIN16U) = [4,8](MBIN16U), 201 5 8 4 8
MOVE_MBIN8_MBIN8_IBIN32S [5,10](MPTR32) = [4,10](MPTR32), 4 5 10 4 10 20
MOVE_MBIN16U_MBIN16U [5,30](MBIN16U) = [4,30](MBIN16U), 201 5 30 4 30
MOVE_MBIN32U_MBIN32U [5,32](MBIN32U) = [4,32](MBIN32U), 202 5 32 4 32
RETURN  =  264

BLOCK: 0x3718670 (2)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x3cbde80 (1)
Preds:
Succs: 2 3
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [5,0](MBIN16S) = [4,8,0](MPTR32), 248 5 0 4 8 134219268 -1 0 14

BLOCK: 0x3cbec70 (2)
Preds: 1
Succs: 4
MOVE_MBIN32S_IBIN32S [5,4](MBIN32S) =  3 5 4 0
BRANCH  =  95 6

BLOCK: 0x3cbfae0 (3)
Preds: 1
Succs: 4
MOVE_MBIN32U_MBIN32U [5,4](MBIN32U) = [4,24](MBIN32U), 202 5 4 4 24

BLOCK: 0x3cc08d0 (4)
Preds: 3 2
Succs: 5 6
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [5,8](MBIN16S) = [4,8,1](MPTR32), 248 5 8 4 8 134219268 -1 1 15

BLOCK: 0x36b7a90 (5)
Preds: 4
Succs: 7
FILL_MEM_BYTES [5,10](MPTR32)[5,12](MPTR32)[5,14](MPTR32) =  115 5 10 20 0
BRANCH  =  95 7

BLOCK: 0x36b8900 (6)
Preds: 4
Succs: 7
MOVE_MBIN8_MBIN8_IBIN32S [5,10](MPTR32) = [4,32](MPTR32), 4 5 10 4 32 20

BLOCK: 0x36b96f0 (7)
Preds: 6 5
Succs: 8 9
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [5,30](MBIN16S) = [4,8,2](MPTR32), 248 5 30 4 8 134219268 -1 2 13

BLOCK: 0x36ba560 (8)
Preds: 7
Succs:
MOVE_MBIN32S_IBIN32S [5,32](MBIN32S) =  3 5 32 0
RETURN  =  264

BLOCK: 0x36bb3d0 (9)
Preds: 7
Succs:
MOVE_MBIN32U_MBIN32U [5,32](MBIN32U) = [4,28](MBIN32U), 202 5 32 4 28
RETURN  =  264

BLOCK: 0x36bc1c0 (10)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x3ac86a0 (1)
Preds:
Succs:
MOVE_MBIN8_MBIN8_IBIN32S [5,0](MPTR32) = [4,0](MPTR32), 4 5 0 4 0 36
RETURN  =  264

BLOCK: 0x3ac9490 (2)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x494d170 (1)
Preds:
Succs: 2 3
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,8](MBIN32S) = [4,8,0](MPTR32),[5,0](MBIN16S), 304 2 8 4 8 0 5 0 -1 2020 12

BLOCK: 0x494df60 (2)
Preds: 1
Succs: 4
BRANCH  =  95 8

BLOCK: 0x494edd0 (3)
Preds: 1
Succs: 4
EQ_MBIN32S_MBIN32S_MBIN32S [2,8](MBIN32S) = [4,24](MBIN32S),[5,4](MBIN32S), 36 2 8 4 24 5 4

BLOCK: 0x494fbc0 (4)
Preds: 3 2
Succs: 5 13
BRANCH_AND [2,4](MBIN32S) = [2,8](MBIN32S), 168 67 0 0 0 2 4 2 8

BLOCK: 0x4950a30 (5)
Preds: 4
Succs: 6 7
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,12](MBIN32S) = [4,8,1](MPTR32),[5,8](MBIN16S), 304 2 12 4 8 1 5 8 -1 2020 12

BLOCK: 0x49518a0 (6)
Preds: 5
Succs: 8
BRANCH  =  95 9

BLOCK: 0x4952710 (7)
Preds: 5
Succs: 8
EQ_MBIN32S_MASCII_MASCII [2,12](MBIN32S) = [4,32](MASCII),[5,10](MASCII), 117 2 12 4 32 5 10 20

BLOCK: 0x4953500 (8)
Preds: 7 6
Succs: 9 13
BRANCH_AND [2,0](MBIN32S) = [2,12](MBIN32S), 168 37 0 0 0 2 0 2 12

BLOCK: 0x4954370 (9)
Preds: 8
Succs: 10 11
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,16](MBIN32S) = [4,8,2](MPTR32),[5,30](MBIN16S), 304 2 16 4 8 2 5 30 -1 2020 12

BLOCK: 0x3724610 (10)
Preds: 9
Succs: 12
BRANCH  =  95 8

BLOCK: 0x3725480 (11)
Preds: 9
Succs: 12
EQ_MBIN32S_MBIN32S_MBIN32S [2,16](MBIN32S) = [4,28](MBIN32S),[5,32](MBIN32S), 36 2 16 4 28 5 32

BLOCK: 0x3726360 (12)
Preds: 11 10
Succs: 13 14
BRANCH_OR [2,0](MBIN32S) = [2,16](MBIN32S), 169 10 0 0 0 2 0 2 16

BLOCK: 0x3727140 (13)
Preds: 12 4 8
Succs:
RETURN_IBIN32S  =  265 0

BLOCK: 0x3727fb0 (14)
Preds: 12
Succs:
RETURN_IBIN32S  =  265 1

BLOCK: 0x3728da0 (15)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x3a109c0 (1)
Preds:
Succs: 2 3
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,8](MBIN32S) = [4,0](MBIN16S),[5,0](MBIN16S), 304 2 8 4 0 -1 5 0 -1 2020 12

BLOCK: 0x3a117b0 (2)
Preds: 1
Succs: 4
BRANCH  =  95 8

BLOCK: 0x3a12620 (3)
Preds: 1
Succs: 4
EQ_MBIN32S_MBIN32S_MBIN32S [2,8](MBIN32S) = [4,4](MBIN32S),[5,4](MBIN32S), 36 2 8 4 4 5 4

BLOCK: 0x3a13410 (4)
Preds: 3 2
Succs: 5 13
BRANCH_AND [2,4](MBIN32S) = [2,8](MBIN32S), 168 67 0 0 0 2 4 2 8

BLOCK: 0x3a14280 (5)
Preds: 4
Succs: 6 7
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,12](MBIN32S) = [4,8](MBIN16S),[5,8](MBIN16S), 304 2 12 4 8 -1 5 8 -1 2020 12

BLOCK: 0x3a150f0 (6)
Preds: 5
Succs: 8
BRANCH  =  95 9

BLOCK: 0x3a15f60 (7)
Preds: 5
Succs: 8
EQ_MBIN32S_MASCII_MASCII [2,12](MBIN32S) = [4,10](MASCII),[5,10](MASCII), 117 2 12 4 10 5 10 20

BLOCK: 0x2b15880 (8)
Preds: 7 6
Succs: 9 13
BRANCH_AND [2,0](MBIN32S) = [2,12](MBIN32S), 168 37 0 0 0 2 0 2 12

BLOCK: 0x2b16690 (9)
Preds: 8
Succs: 10 11
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,16](MBIN32S) = [4,30](MBIN16S),[5,30](MBIN16S), 304 2 16 4 30 -1 5 30 -1 2020 12

BLOCK: 0x2b17500 (10)
Preds: 9
Succs: 12
BRANCH  =  95 8

BLOCK: 0x2b18370 (11)
Preds: 9
Succs: 12
EQ_MBIN32S_MBIN32S_MBIN32S [2,16](MBIN32S) = [4,32](MBIN32S),[5,32](MBIN32S), 36 2 16 4 32 5 32

BLOCK: 0x2b19200 (12)
Preds: 11 10
Succs: 13 14
BRANCH_OR [2,0](MBIN32S) = [2,16](MBIN32S), 169 10 0 0 0 2 0 2 16

BLOCK: 0x2b19fe0 (13)
Preds: 12 4 8
Succs:
RETURN_IBIN32S  =  265 0

BLOCK: 0x2b1ae50 (14)
Preds: 12
Succs:
RETURN_IBIN32S  =  265 1

BLOCK: 0x2b1bc40 (15)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x2b236b0 (1)
Preds:
Succs: 2 3
HDR_MPTR32_IBIN32S_IBIN32S_IBIN32S_IBIN32S_IBIN32S [4,0](MPTR32)[4,4](MBIN16S) =  303 4 0 16 16 4 8 4
MOVE_MBIN64S_MBIN64S [4,16](MBIN64S) = [5,0](MBIN64S), 203 4 16 5 0
OPDATA_MPTR32_IBIN32S [4,8](MPTR32) =  0 4 8 0
OPDATA_MBIN16U_IBIN32S  = [5,8](MBIN16U), 1 5 8 1
OPDATA_MPTR32_IBIN32S [4,24](MPTR32) =  0 4 24 10
OPDATA_MPTR32_IBIN32S  = [5,12](MPTR32), 0 5 12 11
CLAUSE_EVAL  =  108 1448711376 32524 1
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [4,8,1](MPTR32) = [5,16](MBIN16S), 248 4 8 5 16 134218758 1 -1 34

BLOCK: 0x4a4c1f0 (2)
Preds: 1
Succs:
FILL_MEM_BYTES [4,32](MPTR32)[4,34](MPTR32)[4,36](MPTR32) =  115 4 32 20 0
OPDATA_MPTR32_IBIN32S [4,8](MPTR32) =  0 4 8 0
OPDATA_MBIN16U_IBIN32S  = [5,38](MBIN16U), 1 5 38 1
OPDATA_MPTR32_IBIN32S [4,28](MPTR32) =  0 4 28 10
OPDATA_MPTR32_IBIN32S  = [5,40](MPTR32), 0 5 40 11
CLAUSE_EVAL  =  108 1448713768 32524 1
RETURN  =  264

BLOCK: 0x4a4d060 (3)
Preds: 1
Succs:
MOVE_MBIN8_MBIN8_IBIN32S [4,32](MPTR32) = [5,18](MPTR32), 4 4 32 5 18 20
OPDATA_MPTR32_IBIN32S [4,8](MPTR32) =  0 4 8 0
OPDATA_MBIN16U_IBIN32S  = [5,38](MBIN16U), 1 5 38 1
OPDATA_MPTR32_IBIN32S [4,28](MPTR32) =  0 4 28 10
OPDATA_MPTR32_IBIN32S  = [5,40](MPTR32), 0 5 40 11
CLAUSE_EVAL  =  108 1448713768 32524 1
RETURN  =  264

BLOCK: 0x4a4de50 (4)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x4a589b0 (1)
Preds:
Succs:
CONVVCPTR_MBIN64S_MATTR5_IBIN32S [2,0](MBIN64S) = [5,2](MATTR5), 330 2 0 5 2 -1 8 512 8
ENCODE_MASCII_MBIN64S_IBIN32S [4,0](MBIN64S) = [2,0](MBIN64S), 91 4 0 2 0 0
RETURN  =  264

BLOCK: 0x4a597a0 (2)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x4a904e0 (1)
Preds:
Succs: 2 3
HDR_MPTR32_IBIN32S_IBIN32S_IBIN32S_IBIN32S_IBIN32S [4,0](MPTR32)[4,4](MBIN16S) =  303 4 0 12 12 4 8 4
OPDATA_MPTR32_IBIN32S [4,12](MPTR32) =  0 4 12 10
OPDATA_MPTR32_IBIN32S  = [5,0](MPTR32), 0 5 0 11
CLAUSE_EVAL  =  108 1448757280 32524 0
NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S [4,8,0](MPTR32) = [5,4](MBIN16S), 248 4 8 5 4 134218758 0 -1 34

BLOCK: 0x4a912d0 (2)
Preds: 1
Succs:
FILL_MEM_BYTES [4,20](MPTR32)[4,22](MPTR32)[4,24](MPTR32) =  115 4 20 20 0
OPDATA_MPTR32_IBIN32S [4,8](MPTR32) =  0 4 8 0
OPDATA_MBIN16U_IBIN32S  = [5,26](MBIN16U), 1 5 26 1
OPDATA_MPTR32_IBIN32S [4,16](MPTR32) =  0 4 16 10
OPDATA_MPTR32_IBIN32S  = [5,28](MPTR32), 0 5 28 11
CLAUSE_EVAL  =  108 1448759672 32524 1
RETURN  =  264

BLOCK: 0x4a92140 (3)
Preds: 1
Succs:
MOVE_MBIN8_MBIN8_IBIN32S [4,20](MPTR32) = [5,6](MPTR32), 4 4 20 5 6 20
OPDATA_MPTR32_IBIN32S [4,8](MPTR32) =  0 4 8 0
OPDATA_MBIN16U_IBIN32S  = [5,26](MBIN16U), 1 5 26 1
OPDATA_MPTR32_IBIN32S [4,16](MPTR32) =  0 4 16 10
OPDATA_MPTR32_IBIN32S  = [5,28](MPTR32), 0 5 28 11
CLAUSE_EVAL  =  108 1448759672 32524 1
RETURN  =  264

BLOCK: 0x4a92f30 (4)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x514c850 (1)
Preds:
Succs: 2 3
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,4](MBIN32S) = [5,8,0](MPTR32),[4,8](MBIN16S), 304 2 4 5 8 0 4 8 -1 2020 12

BLOCK: 0x514d640 (2)
Preds: 1
Succs: 4
BRANCH  =  95 9

BLOCK: 0x514e4b0 (3)
Preds: 1
Succs: 4
EQ_MBIN32S_MASCII_MASCII [2,4](MBIN32S) = [5,20](MASCII),[4,10](MASCII), 117 2 4 5 20 4 10 20

BLOCK: 0x514f2a0 (4)
Preds: 3 2
Succs: 5 9
BRANCH_AND [2,0](MBIN32S) = [2,4](MBIN32S), 168 37 0 0 0 2 0 2 4

BLOCK: 0x5150110 (5)
Preds: 4
Succs: 6 7
NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S [2,8](MBIN32S) = [5,8,1](MPTR32),[4,30](MBIN16S), 304 2 8 5 8 1 4 30 -1 2020 12

BLOCK: 0x4a632b0 (6)
Preds: 5
Succs: 8
BRANCH  =  95 8

BLOCK: 0x4a640f0 (7)
Preds: 5
Succs: 8
EQ_MBIN32S_MBIN32S_MBIN32S [2,8](MBIN32S) = [5,16](MBIN32S),[4,32](MBIN32S), 36 2 8 5 16 4 32

BLOCK: 0x4a64ee0 (8)
Preds: 7 6
Succs: 9 10
BRANCH_OR [2,0](MBIN32S) = [2,8](MBIN32S), 169 10 0 0 0 2 0 2 8

BLOCK: 0x4a65d50 (9)
Preds: 8 4
Succs:
RETURN_IBIN32S  =  265 0

BLOCK: 0x4a66bc0 (10)
Preds: 8
Succs:
RETURN_IBIN32S  =  265 1

BLOCK: 0x4a679b0 (11)
Preds:
Succs:
RETURN  =  264


-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*

*-*-*-*
CFG [0]
*-*-*-*
BLOCK: 0x4054930 (1)
Preds:
Succs:
OPDATA_MBIN16U_IBIN32S  = [4,0](MBIN16U), 1 4 0 1
OPDATA_MPTR32_IBIN32S [2,0](MPTR32) =  0 2 0 10
OPDATA_MPTR32_IBIN32S  = [4,4](MPTR32), 0 4 4 11
OPDATA_MPTR32_IBIN32S  = [6,0](MPTR32), 0 6 0 12
CLAUSE_EVAL  =  108 1448752192 32524 1
ENCODE_MASCII_MBIN32S_IBIN32S [5,0](MBIN32S) = [2,0](MBIN32S), 89 5 0 2 0 0
RETURN  =  264

BLOCK: 0x4055720 (2)
Preds:
Succs:
RETURN  =  264


MASTER Executor fragment
========================

Fragment ID: 0, Length: 67376

Contents of EX_ROOT [5]:
------------------------

For ComTdb :
Class Version = 1, Class Size = 536
InitialQueueSizeDown = 4, InitialQueueSizeUp = 4
queueResizeLimit = 9, queueResizeFactor = 4
queueSizeDown = 0, queueSizeUp = 0, numBuffers = 0, bufferSize = 0
estimatedRowUsed = 0, estimatedRowsAccessed = 0, expressionMode = 0
Flag = 0000000100101001

For ComTdbRoot :
FirstNRows = -1, baseTablenamePosition = -1
queryType_ = 2, planVersion_ = 2600
rtFlags1_ = 00100000001100000000101000000000
rtFlags2_ = 00000000000000000000000000001000
rtFlags3_ = 0000000000000000
rtFlags4_ = 00000000000000000010000000000000
rtFlags5_ = 00000000000000000000000000000000
queryType_ = 2
inputVarsSize_ = 0
explain_plan_size = 5496

# of Expressions = 4

Expression: inputExpr_ is NULL

Expression: outputExpr_
Expr Len: 496, Consts Len: 0
flags_ = 0000000010001000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000010
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 173
      Text = cast

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 2
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 14
      Text = TRAFODION.SEABASE.T022.ENO

  Clause #2: ex_inout_clause
    OperatorTypeEnum = ITM_NATYPE(2454), NumOperands = 1
    ex_clause::flags_ = 0000000010100000
    ex_inout_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 174
      Text = INTEGER SIGNED

  Clause #3: ex_inout_clause
    OperatorTypeEnum = ITM_NATYPE(2454), NumOperands = 1
    ex_clause::flags_ = 0000000010100100
    ex_inout_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 2
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 175
      Text = CHAR(20) CHARACTER SET ISO88591

  Clause #4: ex_inout_clause
    OperatorTypeEnum = ITM_NATYPE(2454), NumOperands = 1
    ex_clause::flags_ = 0000000010100100
    ex_inout_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 2
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 176
      Text = INTEGER SIGNED

  PCode:

PCode Expr Length: 176
    [1]
    NULL_VIOLATION_MBIN16U (21) 4 0
    MOVE_MBIN32U_MBIN32U (202) 2 0 4 4
    OPDATA_MPTR32_IBIN32S (0) 2 0 10
    CLAUSE_EVAL (108) 1448768272 32524 0
    OPDATA_MPTR32_IBIN32S (0) 4 8 0
    OPDATA_MPTR32_IBIN32S (0) 4 10 10
    CLAUSE_EVAL (108) 1448768824 32524 0
    OPDATA_MPTR32_IBIN32S (0) 4 30 0
    OPDATA_MPTR32_IBIN32S (0) 4 32 10
    CLAUSE_EVAL (108) 1448769384 32524 0

Expression: pkeyExpr_ is NULL

Expression: predExpr_ is NULL

Contents of EX_ONLJ [4]:
------------------------

For ComTdb :
Class Version = 1, Class Size = 264
InitialQueueSizeDown = 4, InitialQueueSizeUp = 4
queueResizeLimit = 9, queueResizeFactor = 4
queueSizeDown = 2048, queueSizeUp = 2048, numBuffers = 5, bufferSize = 5120
estimatedRowUsed = 2, estimatedRowsAccessed = 0, expressionMode = 0
Flag = 0000000000001001

For ComTdbOnlj :
Flags = 0000000000000000, ljRecLen = 0, instantiatedRowAtpIndex = 0

# of Expressions = 4

Expression: postJoinPred_ is NULL

Expression: preJoinPred_ is NULL

Expression: ljExpr_ is NULL

Expression: niExpr_ is NULL

Contents of EX_HASH_GRBY [2]:
-----------------------------

For ComTdb :
Class Version = 1, Class Size = 360
InitialQueueSizeDown = 4, InitialQueueSizeUp = 2048
queueResizeLimit = 9, queueResizeFactor = 4
queueSizeDown = 2048, queueSizeUp = 2048, numBuffers = 5, bufferSize = 262144
estimatedRowUsed = 2, estimatedRowsAccessed = 0, expressionMode = 0
Flag = 0000000000001001

For ComTdbHashGrby :
resultRowLength = 36, extGroupedRowLength = 52
keyLength = 34, isPartialGroup = 0, initialHashTableSize = 262144
memUsagePercent = 5, pressureThreshold = 100, minBuffersToFlush = 4
logDiagnostics = 0
memoryQuotaMB = 1200

# of Expressions = 13

Expression: hashExpr_
Expr Len: 616, Consts Len: 12
flags_ = 0000000010001000
  Clause #1: ex_function_clause
    OperatorTypeEnum = ITM_HASH(2115), NumOperands = 2
    ex_clause::flags_ = 0000000010000010
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 134
      Text = Hash

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 24, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 14
      Text = TRAFODION.SEABASE.T022.ENO

  Clause #2: ex_function_clause
    OperatorTypeEnum = ITM_HASH(2115), NumOperands = 2
    ex_clause::flags_ = 0000000010000010
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 135
      Text = Hash

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 32, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 16, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 15
      Text = TRAFODION.SEABASE.T022.ENAME

  Clause #3: ex_function_clause
    OperatorTypeEnum = ITM_HASHCOMB(2362), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 136
      Text = HashComb

    Operand #1:
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 134
      Text = Hash

    Operand #2:
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 135
      Text = Hash

  Clause #4: ex_function_clause
    OperatorTypeEnum = ITM_HASH(2115), NumOperands = 2
    ex_clause::flags_ = 0000000010000010
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 137
      Text = Hash

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 28, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 12, VoaOffset = -1, NullBitIdx = 2
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 16
      Text = TRAFODION.SEABASE.T022.DNO

  Clause #5: ex_function_clause
    OperatorTypeEnum = ITM_HASHCOMB(2362), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 138
      Text = HashComb

    Operand #1:
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 136
      Text = HashComb

    Operand #2:
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 137
      Text = Hash

  Clause #6: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 139
      Text = cast

    Operand #1:
      Datatype = REC_BIN32_UNSIGNED(133), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 138
      Text = HashComb

  PCode:

PCode Expr Length: 308
    [1]
    NNB_MATTR3_IBIN32S (306) 4 8 0 10  (Tgt: 3)

    [2]
    MOVE_MBIN32S_IBIN32S (3) 2 8 666654765
    BRANCH (95) 8  (Tgt: 4)

    [3]  (Preds: 1 )
    HASH_MBIN32U_MBIN32_IBIN32S_IBIN32S (170) 2 8 4 24 0 4

    [4]  (Preds: 2 )
    NNB_MATTR3_IBIN32S (306) 4 8 1 10  (Tgt: 6)

    [5]
    MOVE_MBIN32S_IBIN32S (3) 2 12 666654765
    BRANCH (95) 8  (Tgt: 7)

    [6]  (Preds: 4 )
    HASH_MBIN32U_MPTR32_IBIN32S_IBIN32S (94) 2 12 4 32 0 20

    [7]  (Preds: 5 )
    HASHCOMB_MBIN32U_MBIN32U_MBIN32U (156) 2 4 2 8 2 12
    NNB_MATTR3_IBIN32S (306) 4 8 2 12  (Tgt: 9)

    [8]
    HASHCOMB_MBIN32U_MBIN32U_MBIN32U (156) 5 0 2 4 1 8
    RETURN (264)

    [9]  (Preds: 7 )
    HASH_MBIN32U_MBIN32_IBIN32S_IBIN32S (170) 2 16 4 28 0 4
    HASHCOMB_MBIN32U_MBIN32U_MBIN32U (156) 5 0 2 4 2 16
    RETURN (264)

Expression: bitMuxExpr_ is NULL

Expression: bitMuxAggrExpr_ is NULL

Expression: hbMoveInExpr_
Expr Len: 736, Consts Len: 175
flags_ = 0000001110011000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 111
      Text = convert

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 24, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 14
      Text = TRAFODION.SEABASE.T022.ENO

  Clause #2: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 113
      Text = convert

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 32, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 16, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 15
      Text = TRAFODION.SEABASE.T022.ENAME

  Clause #3: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 115
      Text = convert

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 28, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 12, VoaOffset = -1, NullBitIdx = 2
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 16
      Text = TRAFODION.SEABASE.T022.DNO

  PCode:

PCode Expr Length: 272
    [1]
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 5 0 4 8 134219268 -1 0 14  (Tgt: 3)

    [2]
    MOVE_MBIN32S_IBIN32S (3) 5 4 0
    BRANCH (95) 6  (Tgt: 4)

    [3]  (Preds: 1 )
    MOVE_MBIN32U_MBIN32U (202) 5 4 4 24

    [4]  (Preds: 2 )
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 5 8 4 8 134219268 -1 1 15  (Tgt: 6)

    [5]
    FILL_MEM_BYTES (115) 5 10 20 0
    BRANCH (95) 7  (Tgt: 7)

    [6]  (Preds: 4 )
    MOVE_MBIN8_MBIN8_IBIN32S (4) 5 10 4 32 20

    [7]  (Preds: 5 )
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 5 30 4 8 134219268 -1 2 13  (Tgt: 9)

    [8]
    MOVE_MBIN32S_IBIN32S (3) 5 32 0
    RETURN (264)

    [9]  (Preds: 7 )
    MOVE_MBIN32U_MBIN32U (202) 5 32 4 28
    RETURN (264)

Native Expr (Length: 167 bytes)
        mov 0x30(%rdx), %rax       488b4230
        mov 0x10(%rax), %rax       488b4010
        mov 0x38(%rsi), %rcx       488b4e38
        mov 0x10(%rcx), %rcx       488b4910
        cmp $0x0, 0x8(%rcx)        80790800
        js 0x2a                    0f8810000000
        mov $0x0, (%rax)           66c7000000
        mov 0x18(%rcx), %edx       8b5118
        mov %edx, 0x4(%rax)        895004
        jmp 0x36                   e90c000000
        mov $0xffff, (%rax)        66c700ffff
        mov $0x0, 0x4(%rax)        c7400400000000
        test $0x40, 0x8(%rcx)      f6410840
        jz 0x62                    0f8422000000
        mov $0xffff, 0x8(%rax)     66c74008ffff
        mov $0x0, 0xa(%rax)        48c7400a00000000
        mov $0x0, 0x12(%rax)       48c7401200000000
        mov $0x0, 0xa(%rax)        c7400a00000000
        jmp 0x7e                   e91c000000
        mov $0x0, 0x8(%rax)        66c740080000
        mov 0x20(%rcx), %rdx       488b5120
        mov %rdx, 0xa(%rax)        4889500a
        mov 0x28(%rcx), %rdx       488b5128
        mov %rdx, 0x12(%rax)       48895012
        mov 0x30(%rcx), %edx       8b5130
        mov %edx, 0x1a(%rax)       89501a
        test $0x20, 0x8(%rcx)      f6410820
        jz 0x98                    0f8410000000
        mov $0xffff, 0x1e(%rax)    66c7401effff
        mov $0x0, 0x20(%rax)       c7402000000000
        xor %eax, %eax             31c0
        ret                        c3
        mov $0x0, 0x1e(%rax)       66c7401e0000
        mov 0x1c(%rcx), %ecx       8b491c
        mov %ecx, 0x20(%rax)       894820
        xor %eax, %eax             31c0
        ret                        c3

Expression: ofMoveInExpr_
Expr Len: 528, Consts Len: 81
flags_ = 0000001110011000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 149
      Text = convert

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 111
      Text = convert

  Clause #2: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 151
      Text = convert

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 113
      Text = convert

  Clause #3: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 153
      Text = convert

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 115
      Text = convert

  PCode:

PCode Expr Length: 152
    [1]
    MOVE_MBIN32U_MBIN32U (202) 5 0 4 0
    MOVE_MBIN32U_MBIN32U (202) 5 4 4 4
    MOVE_MBIN16U_MBIN16U (201) 5 8 4 8
    MOVE_MBIN8_MBIN8_IBIN32S (4) 5 10 4 10 20
    MOVE_MBIN16U_MBIN16U (201) 5 30 4 30
    MOVE_MBIN32U_MBIN32U (202) 5 32 4 32
    RETURN (264)

Native Expr (Length: 73 bytes)
        mov 0x30(%rdx), %rax       488b4230
        mov 0x10(%rax), %rax       488b4010
        mov 0x38(%rsi), %rcx       488b4e38
        mov 0x10(%rcx), %rcx       488b4910
        mov (%rcx), %edx           8b11
        mov %edx, (%rax)           8910
        mov 0x4(%rcx), %edx        8b5104
        mov %edx, 0x4(%rax)        895004
        mov 0x8(%rcx), %dx         668b5108
        mov %dx, 0x8(%rax)         66895008
        mov 0xa(%rcx), %rdx        488b510a
        mov %rdx, 0xa(%rax)        4889500a
        mov 0x12(%rcx), %rdx       488b5112
        mov %rdx, 0x12(%rax)       48895012
        mov 0x1a(%rcx), %edx       8b511a
        mov %edx, 0x1a(%rax)       89501a
        mov 0x1e(%rcx), %dx        668b511e
        mov %dx, 0x1e(%rax)        6689501e
        mov 0x20(%rcx), %ecx       8b4920
        mov %ecx, 0x20(%rax)       894820
        xor %eax, %eax             31c0
        ret                        c3

Expression: resMoveInExpr_
Expr Len: 736, Consts Len: 175
flags_ = 0000001110011000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 5
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 131
      Text = convert

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 24, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 14
      Text = TRAFODION.SEABASE.T022.ENO

  Clause #2: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 5
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 132
      Text = convert

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 32, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 16, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 15
      Text = TRAFODION.SEABASE.T022.ENAME

  Clause #3: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 5
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 133
      Text = convert

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 28, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 12, VoaOffset = -1, NullBitIdx = 2
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 16
      Text = TRAFODION.SEABASE.T022.DNO

  PCode:

PCode Expr Length: 272
    [1]
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 5 0 4 8 134219268 -1 0 14  (Tgt: 3)

    [2]
    MOVE_MBIN32S_IBIN32S (3) 5 4 0
    BRANCH (95) 6  (Tgt: 4)

    [3]  (Preds: 1 )
    MOVE_MBIN32U_MBIN32U (202) 5 4 4 24

    [4]  (Preds: 2 )
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 5 8 4 8 134219268 -1 1 15  (Tgt: 6)

    [5]
    FILL_MEM_BYTES (115) 5 10 20 0
    BRANCH (95) 7  (Tgt: 7)

    [6]  (Preds: 4 )
    MOVE_MBIN8_MBIN8_IBIN32S (4) 5 10 4 32 20

    [7]  (Preds: 5 )
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 5 30 4 8 134219268 -1 2 13  (Tgt: 9)

    [8]
    MOVE_MBIN32S_IBIN32S (3) 5 32 0
    RETURN (264)

    [9]  (Preds: 7 )
    MOVE_MBIN32U_MBIN32U (202) 5 32 4 28
    RETURN (264)

Native Expr (Length: 167 bytes)
        mov 0x48(%rdx), %rax       488b4248
        mov 0x10(%rax), %rax       488b4010
        mov 0x38(%rsi), %rcx       488b4e38
        mov 0x10(%rcx), %rcx       488b4910
        cmp $0x0, 0x8(%rcx)        80790800
        js 0x2a                    0f8810000000
        mov $0x0, (%rax)           66c7000000
        mov 0x18(%rcx), %edx       8b5118
        mov %edx, 0x4(%rax)        895004
        jmp 0x36                   e90c000000
        mov $0xffff, (%rax)        66c700ffff
        mov $0x0, 0x4(%rax)        c7400400000000
        test $0x40, 0x8(%rcx)      f6410840
        jz 0x62                    0f8422000000
        mov $0xffff, 0x8(%rax)     66c74008ffff
        mov $0x0, 0xa(%rax)        48c7400a00000000
        mov $0x0, 0x12(%rax)       48c7401200000000
        mov $0x0, 0xa(%rax)        c7400a00000000
        jmp 0x7e                   e91c000000
        mov $0x0, 0x8(%rax)        66c740080000
        mov 0x20(%rcx), %rdx       488b5120
        mov %rdx, 0xa(%rax)        4889500a
        mov 0x28(%rcx), %rdx       488b5128
        mov %rdx, 0x12(%rax)       48895012
        mov 0x30(%rcx), %edx       8b5130
        mov %edx, 0x1a(%rax)       89501a
        test $0x20, 0x8(%rcx)      f6410820
        jz 0x98                    0f8410000000
        mov $0xffff, 0x1e(%rax)    66c7401effff
        mov $0x0, 0x20(%rax)       c7402000000000
        xor %eax, %eax             31c0
        ret                        c3
        mov $0x0, 0x1e(%rax)       66c7401e0000
        mov 0x1c(%rcx), %ecx       8b491c
        mov %ecx, 0x20(%rax)       894820
        xor %eax, %eax             31c0
        ret                        c3

Expression: hbAggrExpr_ is NULL

Expression: ofAggrExpr_ is NULL

Expression: resAggrExpr_ is NULL

Expression: havingExpr_ is NULL

Expression: moveOutExpr_
Expr Len: 448, Consts Len: 99
flags_ = 0000001110011000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_CONVERT(2451), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 36, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 5
      Offset = 0, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 148
      Text = convert

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 36, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 0, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 147
      Text = CHAR(36) CHARACTER SET ISO88591

  PCode:

PCode Expr Length: 52
    [1]
    MOVE_MBIN8_MBIN8_IBIN32S (4) 5 0 4 0 36
    RETURN (264)

Native Expr (Length: 91 bytes)
        mov 0x30(%rdx), %rax       488b4230
        mov 0x48(%rdx), %rcx       488b4a48
        mov 0x10(%rax), %rdi       488b7810
        mov 0x10(%rcx), %rcx       488b4910
        mov $0x0, -0x8(%rsp)       48c74424f800000000
        xor %edx, %edx             31d2
        mov (%rdi,%rdx), %rsi      488b3417
        mov %rsi, (%rcx,%rdx)      48893411
        add $0x8, %rdx             4883c208
        mov %rdx, -0x8(%rsp)       48895424f8
        cmp $0x20, %rdx            4883fa20
        jb 0x1b                    0f82e5ffffff
        cmp $0x23, %rdx            4883fa23
        ja 0x58                    0f8718000000
        mov (%rdi,%rdx), %al       8a0417
        mov %al, (%rcx,%rdx)       880411
        inc %rdx                   48ffc2
        mov %rdx, -0x8(%rsp)       48895424f8
        cmp $0x24, %rdx            4883fa24
        jb 0x40                    0f82e8ffffff
        xor %eax, %eax             31c0
        ret                        c3

Expression: hbSearchExpr_
Expr Len: 1056, Consts Len: 376
flags_ = 0000001110011000
  Clause #1: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 112
      Text = =

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 24, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 14
      Text = TRAFODION.SEABASE.T022.ENO

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 111
      Text = convert

  Clause #2: ex_branch_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 2
    flags_ = 0000000010000000
    PCODE  = supported
    branch to = #4
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

  Clause #3: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 114
      Text = =

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 32, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 16, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 15
      Text = TRAFODION.SEABASE.T022.ENAME

    Operand #2:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 113
      Text = convert

  Clause #4: ex_bool_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_bool_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 141
      Text = and

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 112
      Text = =

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 114
      Text = =

  Clause #5: ex_branch_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 2
    flags_ = 0000000010000000
    PCODE  = supported
    branch to = #7
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

  Clause #6: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 116
      Text = =

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 28, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 12, VoaOffset = -1, NullBitIdx = 2
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 16
      Text = TRAFODION.SEABASE.T022.DNO

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 115
      Text = convert

  Clause #7: ex_bool_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_bool_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 142
      Text = and

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 141
      Text = and

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 116
      Text = =

  Clause #8: bool_result_clause
    OperatorTypeEnum = ITM_BOOL_RESULT(2107), NumOperands = 1
    ex_clause::flags_ = 0000000010000000
    bool_result_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 143
      Text = bool_result

  PCode:

PCode Expr Length: 392
    [1]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 8 4 8 0 5 0 -1 2020 12  (Tgt: 3)

    [2]
    BRANCH (95) 8  (Tgt: 4)

    [3]  (Preds: 1 )
    EQ_MBIN32S_MBIN32S_MBIN32S (36) 2 8 4 24 5 4

    [4]  (Preds: 2 )
    BRANCH_AND (168) 67 0 0 0 2 4 2 8  (Tgt: 13)

    [5]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 12 4 8 1 5 8 -1 2020 12  (Tgt: 7)

    [6]
    BRANCH (95) 9  (Tgt: 8)

    [7]  (Preds: 5 )
    EQ_MBIN32S_MASCII_MASCII (117) 2 12 4 32 5 10 20

    [8]  (Preds: 6 )
    BRANCH_AND (168) 37 0 0 0 2 0 2 12  (Tgt: 13)

    [9]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 16 4 8 2 5 30 -1 2020 12  (Tgt: 11)

    [10]
    BRANCH (95) 8  (Tgt: 12)

    [11]  (Preds: 9 )
    EQ_MBIN32S_MBIN32S_MBIN32S (36) 2 16 4 28 5 32

    [12]  (Preds: 10 )
    BRANCH_OR (169) 10 0 0 0 2 0 2 16  (Tgt: 14)

    [13]  (Preds: 4 8 )
    RETURN_IBIN32S (265) 0

    [14]  (Preds: 12 )
    RETURN_IBIN32S (265) 1

Native Expr (Length: 368 bytes)
        push %rbp                  55
        mov %rsp, %rbp             4889e5
        sub $0x10, %rsp            4883ec10
        mov 0x30(%rdx), %rax       488b4230
        mov 0x10(%rax), %r9        4c8b4810
        movzx (%r9), %eax          410fb701
        mov 0x38(%rsi), %rcx       488b4e38
        mov 0x10(%rcx), %r8        4c8b4110
        mov 0x8(%r8), %sil         418a7008
        test %sil, %sil            4084f6
        js 0x51                    0f8828000000
        test %ax, %ax              6685c0
        jnz 0x5f                   0f852d000000
        mov $0x0, -0x4(%rbp)       c745fc00000000
        mov 0x18(%r8), %eax        418b4018
        mov $0x1, %edi             bf01000000
        cmp 0x4(%r9), %eax         413b4104
        jnz 0x5f                   0f8513000000
        jmp 0x61                   e910000000
        mov $0x1, %edi             bf01000000
        test %ax, %ax              6685c0
        jnz 0x61                   0f8502000000
        xor %edi, %edi             31ff
        mov %edi, -0x4(%rbp)       897dfc
        mov $0x3, %eax             b803000000
        test %edi, %edi            85ff
        jz 0x16b                   0f84fa000000
        movzx 0x8(%r9), %ecx       410fb74908
        test $0x40, %sil           40f6c640
        jz 0x93                    0f8413000000
        mov $0x1, %esi             be01000000
        test %cx, %cx              6685c9
        jnz 0x109                  0f857b000000
        jmp 0x9c                   e909000000
        test %cx, %cx              6685c9
        jz 0xa3                    0f8407000000
        xor %esi, %esi             31f6
        jmp 0x109                  e966000000
        mov $0x0, -0x8(%rbp)       c745f800000000
        mov %rsp, %rcx             4889e1
        lea -0x10(%rcx), %rsi      488d71f0
        mov %rsi, %rsp             4889f4
        mov $0x0, -0x10(%rcx)      48c741f000000000
        xor %edi, %edi             31ff
        mov 0x20(%r8,%rdi), %rcx   498b4c3820
        cmp 0xa(%r9,%rdi), %rcx    493b4c390a
        jnz 0xdf                   0f8511000000
        add $0x8, %rdi             4883c708
        mov %rdi, (%rsi)           48893e
        cmp $0x10, %rdi            4883ff10
        jb 0xbe                    0f82dfffffff
        mov 0xa(%r9,%rdi), %dl     418a54390a
        mov 0x20(%r8,%rdi), %cl    418a4c3820
        cmp %dl, %cl               38d1
        jnz 0x101                  0f8510000000
        inc %rdi                   48ffc7
        mov %rdi, (%rsi)           48893e
        cmp $0x14, %rdi            4883ff14
        jb 0xdf                    0f82deffffff
        cmp %dl, %cl               38d1
        setz %cl                   0f94c1
        movzx %cl, %esi            0fb6f1
        mov %esi, -0x8(%rbp)       8975f8
        test %esi, %esi            85f6
        jz 0x16b                   0f8457000000
        movzx 0x1e(%r9), %ecx      410fb7491e
        test $0x20, 0x8(%r8)       41f6400820
        jz 0x137                   0f8413000000
        mov $0x1, %eax             b801000000
        test %cx, %cx              6685c9
        jnz 0x15c                  0f852a000000
        jmp 0x15a                  e923000000
        test %cx, %cx              6685c9
        jnz 0x15a                  0f851a000000
        mov $0x0, -0xc(%rbp)       c745f400000000
        mov 0x1c(%r8), %ecx        418b481c
        mov $0x1, %eax             b801000000
        cmp 0x20(%r9), %ecx        413b4920
        jz 0x15c                   0f8402000000
        xor %eax, %eax             31c0
        mov %eax, -0xc(%rbp)       8945f4
        cmp $0x1, %eax             83f801
        setnz %al                  0f95c0
        movzx %al, %eax            0fb6c0
        or $0x2, %eax              83c802
        mov %rbp, %rsp             4889ec
        pop %rbp                   5d
        ret                        c3

Expression: ofSearchExpr_
Expr Len: 1064, Consts Len: 377
flags_ = 0000001110011000
  Clause #1: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 150
      Text = =

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 111
      Text = convert

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 149
      Text = convert

  Clause #2: ex_branch_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 2
    flags_ = 0000000010000000
    PCODE  = supported
    branch to = #4
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

  Clause #3: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 152
      Text = =

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 113
      Text = convert

    Operand #2:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 151
      Text = convert

  Clause #4: ex_bool_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_bool_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 155
      Text = and

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 150
      Text = =

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 152
      Text = =

  Clause #5: ex_branch_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 2
    flags_ = 0000000010000000
    PCODE  = supported
    branch to = #7
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

  Clause #6: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 154
      Text = =

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 3
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 115
      Text = convert

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 153
      Text = convert

  Clause #7: ex_bool_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_bool_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 156
      Text = and

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 155
      Text = and

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 154
      Text = =

  Clause #8: bool_result_clause
    OperatorTypeEnum = ITM_BOOL_RESULT(2107), NumOperands = 1
    ex_clause::flags_ = 0000000010000000
    bool_result_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 157
      Text = bool_result

  PCode:

PCode Expr Length: 392
    [1]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 8 4 0 -1 5 0 -1 2020 12  (Tgt: 3)

    [2]
    BRANCH (95) 8  (Tgt: 4)

    [3]  (Preds: 1 )
    EQ_MBIN32S_MBIN32S_MBIN32S (36) 2 8 4 4 5 4

    [4]  (Preds: 2 )
    BRANCH_AND (168) 67 0 0 0 2 4 2 8  (Tgt: 13)

    [5]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 12 4 8 -1 5 8 -1 2020 12  (Tgt: 7)

    [6]
    BRANCH (95) 9  (Tgt: 8)

    [7]  (Preds: 5 )
    EQ_MBIN32S_MASCII_MASCII (117) 2 12 4 10 5 10 20

    [8]  (Preds: 6 )
    BRANCH_AND (168) 37 0 0 0 2 0 2 12  (Tgt: 13)

    [9]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 16 4 30 -1 5 30 -1 2020 12  (Tgt: 11)

    [10]
    BRANCH (95) 8  (Tgt: 12)

    [11]  (Preds: 9 )
    EQ_MBIN32S_MBIN32S_MBIN32S (36) 2 16 4 32 5 32

    [12]  (Preds: 10 )
    BRANCH_OR (169) 10 0 0 0 2 0 2 16  (Tgt: 14)

    [13]  (Preds: 4 8 )
    RETURN_IBIN32S (265) 0

    [14]  (Preds: 12 )
    RETURN_IBIN32S (265) 1

Native Expr (Length: 369 bytes)
        push %rbp                  55
        mov %rsp, %rbp             4889e5
        sub $0x10, %rsp            4883ec10
        mov 0x30(%rdx), %rax       488b4230
        mov 0x10(%rax), %r8        4c8b4010
        movzx (%r8), %eax          410fb700
        mov 0x38(%rsi), %rcx       488b4e38
        mov 0x10(%rcx), %r9        4c8b4910
        cmp $0x0, (%r9)            6641833900
        jz 0x3a                    0f8413000000
        mov $0x1, %esi             be01000000
        test %ax, %ax              6685c0
        jnz 0x5f                   0f852a000000
        jmp 0x5d                   e923000000
        test %ax, %ax              6685c0
        jnz 0x5d                   0f851a000000
        mov $0x0, -0x4(%rbp)       c745fc00000000
        mov 0x4(%r9), %eax         418b4104
        mov $0x1, %esi             be01000000
        cmp 0x4(%r8), %eax         413b4004
        jz 0x5f                    0f8402000000
        xor %esi, %esi             31f6
        mov %esi, -0x4(%rbp)       8975fc
        mov $0x3, %eax             b803000000
        test %esi, %esi            85f6
        jz 0x16c                   0f84fd000000
        movzx 0x8(%r8), %ecx       410fb74808
        cmp $0x0, 0x8(%r9)         664183790800
        jz 0x93                    0f8413000000
        mov $0x1, %esi             be01000000
        test %cx, %cx              6685c9
        jnz 0x109                  0f857b000000
        jmp 0x9c                   e909000000
        test %cx, %cx              6685c9
        jz 0xa3                    0f8407000000
        xor %esi, %esi             31f6
        jmp 0x109                  e966000000
        mov $0x0, -0x8(%rbp)       c745f800000000
        mov %rsp, %rcx             4889e1
        lea -0x10(%rcx), %rsi      488d71f0
        mov %rsi, %rsp             4889f4
        mov $0x0, -0x10(%rcx)      48c741f000000000
        xor %edi, %edi             31ff
        mov 0xa(%r9,%rdi), %rcx    498b4c390a
        cmp 0xa(%r8,%rdi), %rcx    493b4c380a
        jnz 0xdf                   0f8511000000
        add $0x8, %rdi             4883c708
        mov %rdi, (%rsi)           48893e
        cmp $0x10, %rdi            4883ff10
        jb 0xbe                    0f82dfffffff
        mov 0xa(%r8,%rdi), %cl     418a4c380a
        mov 0xa(%r9,%rdi), %dl     418a54390a
        cmp %cl, %dl               38ca
        jnz 0x101                  0f8510000000
        inc %rdi                   48ffc7
        mov %rdi, (%rsi)           48893e
        cmp $0x14, %rdi            4883ff14
        jb 0xdf                    0f82deffffff
        cmp %cl, %dl               38ca
        setz %cl                   0f94c1
        movzx %cl, %esi            0fb6f1
        mov %esi, -0x8(%rbp)       8975f8
        test %esi, %esi            85f6
        jz 0x16c                   0f8458000000
        movzx 0x1e(%r8), %ecx      410fb7481e
        cmp $0x0, 0x1e(%r9)        664183791e00
        jz 0x138                   0f8413000000
        mov $0x1, %eax             b801000000
        test %cx, %cx              6685c9
        jnz 0x15d                  0f852a000000
        jmp 0x15b                  e923000000
        test %cx, %cx              6685c9
        jnz 0x15b                  0f851a000000
        mov $0x0, -0xc(%rbp)       c745f400000000
        mov 0x20(%r9), %ecx        418b4920
        mov $0x1, %eax             b801000000
        cmp 0x20(%r8), %ecx        413b4820
        jz 0x15d                   0f8402000000
        xor %eax, %eax             31c0
        mov %eax, -0xc(%rbp)       8945f4
        cmp $0x1, %eax             83f801
        setnz %al                  0f95c0
        movzx %al, %eax            0fb6c0
        or $0x2, %eax              83c802
        mov %rbp, %rsp             4889ec
        pop %rbp                   5d
        ret                        c3

Contents of EX_TRAF_SELECT [1]:
-------------------------------

For ComTdb :
Class Version = 1, Class Size = 584
InitialQueueSizeDown = 4, InitialQueueSizeUp = 4
queueResizeLimit = 9, queueResizeFactor = 4
queueSizeDown = 2048, queueSizeUp = 2048, numBuffers = 4, bufferSize = 1048648
estimatedRowUsed = 25, estimatedRowsAccessed = 0, expressionMode = 0
Flag = 0000000000001001

For ComTdbHbaseAccess :
accessType_ = SELECT_
accessDetail_ = EX_TRAF_SELECT
tableName_ = TRAFODION.SEABASE.T022
asciiTI_ = 4, convertTI_ = 2, rowIdTI_ = 3, returnedTI_ = 3
rowIdAsciiTI_ = 5, updateTI_ = 0, mergeInsertTI_ = 0
returnedFetchedTI_ = 0, returnedUpdatedTI_ = 0, mergeInsertRowIdTI_ = 0
hbaseTimestampTI_ = 0, hbaseVersionTI_ = 0
asciiRowLen_ = 44, convertRowLen_ = 52, rowIdLen_ = 8, outputRowLen_ = 52
updateRowLen_ = 0, returnFetchedRowLen_ = 0, returnUpdateedRowLen_ = 0
mergeInsertRowLen_ = 0, keyLen_ = 0
Flag = 0000000000000100
server_ = , zkPort_ =

listOfFetchedColNames_(numEntries = 4):

  Entry #1: #1:1
  Entry #2: #1:2
  Entry #3: #1:3
  Entry #4: #1:4

listOfScanRows_(numEntries = 1):

  Entry #1:
    beginRowId_(incl) =
        0:
    endRowId_(incl) =
        0:

    colTS_=-1

# of Expressions = 18

Expression: Convert Expr
Expr Len: 704, Consts Len: 8
flags_ = 0000000010001000
  Clause #1: ex_function_clause
    OperatorTypeEnum = ITM_HEADER(2375), NumOperands = 1
    ex_clause::flags_ = 0000000010000000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 16, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      RelOffset = 0, VoaOffset = -1, NullBitIdx = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 0
      Text = Hdr

  Clause #2: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN64_SIGNED(134), Length = 8, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 16, NullIndOffset = -1, VClenIndOffset = -1
      RelOffset = 0, VoaOffset = -1, NullBitIdx = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 119
      Text = cast

    Operand #1:
      Datatype = REC_BIN64_SIGNED(134), Length = 8, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 118
      Text = LARGEINT SIGNED

  Clause #3: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 121
      Text = cast

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 12, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 120
      Text = INTEGER SIGNED

  Clause #4: ex_function_clause
    OperatorTypeEnum = ITM_COMP_DECODE(2175), NumOperands = 2
    ex_clause::flags_ = 0000000010100110
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 24, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 122
      Text = comp_decode

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 121
      Text = cast

  Clause #5: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 32, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 16, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 124
      Text = cast

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 18, NullIndOffset = 16, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 123
      Text = CHAR(20) CHARACTER SET ISO88591

  Clause #6: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 126
      Text = cast

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 40, NullIndOffset = 38, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 125
      Text = INTEGER SIGNED

  Clause #7: ex_function_clause
    OperatorTypeEnum = ITM_COMP_DECODE(2175), NumOperands = 2
    ex_clause::flags_ = 0000000010100110
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 28, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 12, VoaOffset = -1, NullBitIdx = 2
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 127
      Text = comp_decode

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 12, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 126
      Text = cast

  PCode:

PCode Expr Length: 404
    [1]
    HDR_MPTR32_IBIN32S_IBIN32S_IBIN32S_IBIN32S_IBIN32S (303) 4 0 16 16 4 8 4
    MOVE_MBIN64S_MBIN64S (203) 4 16 5 0
    OPDATA_MPTR32_IBIN32S (0) 4 8 0
    OPDATA_MBIN16U_IBIN32S (1) 5 8 1
    OPDATA_MPTR32_IBIN32S (0) 4 24 10
    OPDATA_MPTR32_IBIN32S (0) 5 12 11
    CLAUSE_EVAL (108) 1448711376 32524 1
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 4 8 5 16 134218758 1 -1 34  (Tgt: 3)

    [2]
    FILL_MEM_BYTES (115) 4 32 20 0
    OPDATA_MPTR32_IBIN32S (0) 4 8 0
    OPDATA_MBIN16U_IBIN32S (1) 5 38 1
    OPDATA_MPTR32_IBIN32S (0) 4 28 10
    OPDATA_MPTR32_IBIN32S (0) 5 40 11
    CLAUSE_EVAL (108) 1448713768 32524 1
    RETURN (264)

    [3]  (Preds: 1 )
    MOVE_MBIN8_MBIN8_IBIN32S (4) 4 32 5 18 20
    OPDATA_MPTR32_IBIN32S (0) 4 8 0
    OPDATA_MBIN16U_IBIN32S (1) 5 38 1
    OPDATA_MPTR32_IBIN32S (0) 4 28 10
    OPDATA_MPTR32_IBIN32S (0) 5 40 11
    CLAUSE_EVAL (108) 1448713768 32524 1
    RETURN (264)

Expression: ScanExpr is NULL

Expression: RowIdExpr
Expr Len: 384, Consts Len: 8
flags_ = 0000000010001000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_conv_clause::flags_ = 0000000000010000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN64_SIGNED(134), Length = 8, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 129
      Text = cast

    Operand #1:
      Datatype = REC_BYTE_V_ASCII(64), Length = 8, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 5
      Offset = 2, NullIndOffset = -1, VClenIndOffset = 0
      NullIndLength = 0, VClenIndLength = 2
      ValueId = 128
      Text = VARCHAR(8) CHARACTER SET ISO88591

  Clause #2: ex_function_clause
    OperatorTypeEnum = ITM_COMP_ENCODE(2114), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 8, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 3
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 130
      Text = comp_encode

    Operand #1:
      Datatype = REC_BIN64_SIGNED(134), Length = 8, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 129
      Text = cast

  PCode:

PCode Expr Length: 88
    [1]
    CONVVCPTR_MBIN64S_MATTR5_IBIN32S (330) 2 0 5 2 -1 8 512 8
    ENCODE_MASCII_MBIN64S_IBIN32S (91) 4 0 2 0 0
    RETURN (264)

Expression: UpdateExpr is NULL

Expression: MergeInsertExpr is NULL

Expression: LowKeyExpr is NULL

Expression: HighKeyExpr is NULL

Expression: ReturnFetchExpr is NULL

Expression: ReturnUpdateExpr is NULL

Expression: ReturnMergeInsertExpr is NULL

Expression: mergeUpdScanExpr is NULL

Expression: mergeInsertRowIdExpr is NULL

Expression: encodedKeyExpr is NULL

Expression: keyColValExpr is NULL

Expression: hbaseFilterExpr is NULL

Expression: preCondExpr is NULL

Expression: insConstraintExpr is NULL

Expression: updConstraintExpr is NULL

Contents of EX_TRAF_KEY_ROWSET_SELECT [3]:
------------------------------------------

For ComTdb :
Class Version = 1, Class Size = 584
InitialQueueSizeDown = 2048, InitialQueueSizeUp = 2048
queueResizeLimit = 9, queueResizeFactor = 4
queueSizeDown = 2048, queueSizeUp = 2048, numBuffers = 4, bufferSize = 1048648
estimatedRowUsed = 1, estimatedRowsAccessed = 0, expressionMode = 0
Flag = 0000000000001001

For ComTdbHbaseAccess :
accessType_ = SELECT_
accessDetail_ = EX_TRAF_KEY_ROWSET_SELECT
tableName_ = TRAFODION.SEABASE.T021
asciiTI_ = 4, convertTI_ = 2, rowIdTI_ = 3, returnedTI_ = 3
rowIdAsciiTI_ = 5, updateTI_ = 0, mergeInsertTI_ = 0
returnedFetchedTI_ = 0, returnedUpdatedTI_ = 0, mergeInsertRowIdTI_ = 0
hbaseTimestampTI_ = 0, hbaseVersionTI_ = 0
asciiRowLen_ = 32, convertRowLen_ = 40, rowIdLen_ = 0, outputRowLen_ = 40
updateRowLen_ = 0, returnFetchedRowLen_ = 0, returnUpdateedRowLen_ = 0
mergeInsertRowLen_ = 0, keyLen_ = 4
Flag = 0000001010000100
server_ = , zkPort_ =

listOfFetchedColNames_(numEntries = 3):

  Entry #1: #1:1
  Entry #2: #1:2
  Entry #3: #1:3

# of Expressions = 18

Expression: Convert Expr
Expr Len: 648, Consts Len: 8
flags_ = 0000000010001000
  Clause #1: ex_function_clause
    OperatorTypeEnum = ITM_HEADER(2375), NumOperands = 1
    ex_clause::flags_ = 0000000010000000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 12, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      RelOffset = 0, VoaOffset = -1, NullBitIdx = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 0
      Text = Hdr

  Clause #2: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 163
      Text = cast

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 162
      Text = INTEGER SIGNED

  Clause #3: ex_function_clause
    OperatorTypeEnum = ITM_COMP_DECODE(2175), NumOperands = 2
    ex_clause::flags_ = 0000000010100000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 12, NullIndOffset = -1, VClenIndOffset = -1
      RelOffset = 0, VoaOffset = -1, NullBitIdx = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 164
      Text = comp_decode

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 163
      Text = cast

  Clause #4: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 20, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 166
      Text = cast

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 6, NullIndOffset = 4, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 165
      Text = CHAR(20) CHARACTER SET ISO88591

  Clause #5: ex_conv_clause
    OperatorTypeEnum = ITM_CAST(2452), NumOperands = 2
    ex_clause::flags_ = 0000000010000110
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = 4, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 168
      Text = cast

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 28, NullIndOffset = 26, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 167
      Text = INTEGER SIGNED

  Clause #6: ex_function_clause
    OperatorTypeEnum = ITM_COMP_DECODE(2175), NumOperands = 2
    ex_clause::flags_ = 0000000010100110
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 16, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 4, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 169
      Text = comp_decode

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = 4, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 168
      Text = cast

  PCode:

PCode Expr Length: 352
    [1]
    HDR_MPTR32_IBIN32S_IBIN32S_IBIN32S_IBIN32S_IBIN32S (303) 4 0 12 12 4 8 4
    OPDATA_MPTR32_IBIN32S (0) 4 12 10
    OPDATA_MPTR32_IBIN32S (0) 5 0 11
    CLAUSE_EVAL (108) 1448757280 32524 0
    NOT_NULL_BRANCH_MBIN32S_MBIN32S_IATTR3_IBIN32S (248) 4 8 5 4 134218758 0 -1 34  (Tgt: 3)

    [2]
    FILL_MEM_BYTES (115) 4 20 20 0
    OPDATA_MPTR32_IBIN32S (0) 4 8 0
    OPDATA_MBIN16U_IBIN32S (1) 5 26 1
    OPDATA_MPTR32_IBIN32S (0) 4 16 10
    OPDATA_MPTR32_IBIN32S (0) 5 28 11
    CLAUSE_EVAL (108) 1448759672 32524 1
    RETURN (264)

    [3]  (Preds: 1 )
    MOVE_MBIN8_MBIN8_IBIN32S (4) 4 20 5 6 20
    OPDATA_MPTR32_IBIN32S (0) 4 8 0
    OPDATA_MBIN16U_IBIN32S (1) 5 26 1
    OPDATA_MPTR32_IBIN32S (0) 4 16 10
    OPDATA_MPTR32_IBIN32S (0) 5 28 11
    CLAUSE_EVAL (108) 1448759672 32524 1
    RETURN (264)

Expression: ScanExpr
Expr Len: 856, Consts Len: 284
flags_ = 0000001110011000
  Clause #1: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 107
      Text = =

    Operand #1:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 20, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 8, VoaOffset = -1, NullBitIdx = 0
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 6
      Text = TRAFODION.SEABASE.T021.ENAME

    Operand #2:
      Datatype = REC_BYTE_F_ASCII(0), Length = 20, Null Flag = 1
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 2
      Offset = 10, NullIndOffset = 8, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 15
      Text = TRAFODION.SEABASE.T022.ENAME

  Clause #2: ex_branch_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 2
    flags_ = 0000000010000000
    PCODE  = supported
    branch to = #4
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0

  Clause #3: ex_comp_clause
    OperatorTypeEnum = ITM_EQUAL(2020), NumOperands = 3
    ex_clause::flags_ = 0000000010000011
    ex_comp_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 108
      Text = =

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000001
      Tuple Data Format = SQLMX_ALIGNED_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 16, NullIndOffset = 8, VClenIndOffset = -1
      RelOffset = 4, VoaOffset = -1, NullBitIdx = 1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 7
      Text = TRAFODION.SEABASE.T021.DNO

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 2
      Offset = 32, NullIndOffset = 30, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 16
      Text = TRAFODION.SEABASE.T022.DNO

  Clause #4: ex_bool_clause
    OperatorTypeEnum = ITM_AND(2001), NumOperands = 3
    ex_clause::flags_ = 0000000010000000
    ex_bool_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 170
      Text = and

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 4, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 107
      Text = =

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 8, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 108
      Text = =

  Clause #5: bool_result_clause
    OperatorTypeEnum = ITM_BOOL_RESULT(2107), NumOperands = 1
    ex_clause::flags_ = 0000000010000000
    bool_result_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 171
      Text = bool_result

  PCode:

PCode Expr Length: 276
    [1]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 4 5 8 0 4 8 -1 2020 12  (Tgt: 3)

    [2]
    BRANCH (95) 9  (Tgt: 4)

    [3]  (Preds: 1 )
    EQ_MBIN32S_MASCII_MASCII (117) 2 4 5 20 4 10 20

    [4]  (Preds: 2 )
    BRANCH_AND (168) 37 0 0 0 2 0 2 4  (Tgt: 9)

    [5]
    NNB_SPECIAL_NULLS_MBIN32S_MATTR3_MATTR3_IBIN32S_IBIN32S (304) 2 8 5 8 1 4 30 -1 2020 12  (Tgt: 7)

    [6]
    BRANCH (95) 8  (Tgt: 8)

    [7]  (Preds: 5 )
    EQ_MBIN32S_MBIN32S_MBIN32S (36) 2 8 5 16 4 32

    [8]  (Preds: 6 )
    BRANCH_OR (169) 10 0 0 0 2 0 2 8  (Tgt: 10)

    [9]  (Preds: 4 )
    RETURN_IBIN32S (265) 0

    [10]  (Preds: 8 )
    RETURN_IBIN32S (265) 1

Native Expr (Length: 276 bytes)
        push %rbp                  55
        mov %rsp, %rbp             4889e5
        sub $0x10, %rsp            4883ec10
        mov 0x30(%rsi), %rax       488b4630
        mov 0x10(%rax), %r8        4c8b4010
        movzx 0x8(%r8), %eax       410fb74008
        mov 0x30(%rdx), %rcx       488b4a30
        mov 0x10(%rcx), %rdi       488b7910
        cmp $0x0, 0x8(%rdi)        807f0800
        js 0x9a                    0f8873000000
        test %ax, %ax              6685c0
        jnz 0xa8                   0f8578000000
        mov $0x0, -0x4(%rbp)       c745fc00000000
        mov %rsp, %rcx             4889e1
        lea -0x10(%rcx), %rax      488d41f0
        mov %rax, %rsp             4889c4
        mov $0x0, -0x10(%rcx)      48c741f000000000
        xor %esi, %esi             31f6
        mov 0x14(%rdi,%rsi), %rcx  488b4c3714
        cmp 0xa(%r8,%rsi), %rcx    493b4c300a
        jnz 0x6c                   0f8511000000
        add $0x8, %rsi             4883c608
        mov %rsi, (%rax)           488930
        cmp $0x10, %rsi            4883fe10
        jb 0x4b                    0f82dfffffff
        mov 0xa(%r8,%rsi), %cl     418a4c300a
        mov 0x14(%rdi,%rsi), %dl   8a543714
        cmp %cl, %dl               38ca
        jnz 0x8d                   0f8510000000
        inc %rsi                   48ffc6
        mov %rsi, (%rax)           488930
        cmp $0x14, %rsi            4883fe14
        jb 0x6c                    0f82dfffffff
        cmp %cl, %dl               38ca
        setz %al                   0f94c0
        movzx %al, %esi            0fb6f0
        jmp 0xaa                   e910000000
        mov $0x1, %esi             be01000000
        test %ax, %ax              6685c0
        jnz 0xaa                   0f8502000000
        xor %esi, %esi             31f6
        mov %esi, -0x4(%rbp)       8975fc
        mov $0x3, %eax             b803000000
        test %esi, %esi            85f6
        jz 0x10f                   0f8455000000
        movzx 0x1e(%r8), %ecx      410fb7481e
        test $0x40, 0x8(%rdi)      f6470840
        jz 0xdc                    0f8413000000
        mov $0x1, %eax             b801000000
        test %cx, %cx              6685c9
        jnz 0x100                  0f8529000000
        jmp 0xfe                   e922000000
        test %cx, %cx              6685c9
        jnz 0xfe                   0f8519000000
        mov $0x0, -0x8(%rbp)       c745f800000000
        mov 0x10(%rdi), %ecx       8b4f10
        mov $0x1, %eax             b801000000
        cmp 0x20(%r8), %ecx        413b4820
        jz 0x100                   0f8402000000
        xor %eax, %eax             31c0
        mov %eax, -0x8(%rbp)       8945f8
        cmp $0x1, %eax             83f801
        setnz %al                  0f95c0
        movzx %al, %eax            0fb6c0
        or $0x2, %eax              83c802
        mov %rbp, %rsp             4889ec
        pop %rbp                   5d
        ret                        c3

Expression: RowIdExpr is NULL

Expression: UpdateExpr is NULL

Expression: MergeInsertExpr is NULL

Expression: LowKeyExpr
Expr Len: 440, Consts Len: 8
flags_ = 0000000010001000
  Clause #1: ex_conv_clause
    OperatorTypeEnum = ITM_NARROW(2350), NumOperands = 3
    ex_clause::flags_ = 0000000010101010
    ex_conv_clause::flags_ = 0000000000000000
    PCODE  = not supported
    Operand #0 (result):
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 160
      Text = narrow

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 1
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000000000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 2
      Offset = 4, NullIndOffset = 0, VClenIndOffset = -1
      NullIndLength = 2, VClenIndLength = 0
      ValueId = 14
      Text = TRAFODION.SEABASE.T022.ENO

    Operand #2:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 1, AtpIndex = 4
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 117
      Text = \:_sys_dataConversionErrorFlag

  Clause #2: ex_function_clause
    OperatorTypeEnum = ITM_COMP_ENCODE(2114), NumOperands = 2
    ex_clause::flags_ = 0000000010000000
    ex_function_clause::flags_ = 0000000000000000
    PCODE  = supported
    Operand #0 (result):
      Datatype = REC_BYTE_F_ASCII(0), Length = 4, Null Flag = 0
      Precision = 0, Scale = 1, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLMX_KEY_FORMAT
      Atp = 1, AtpIndex = 2
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 161
      Text = comp_encode

    Operand #1:
      Datatype = REC_BIN32_SIGNED(132), Length = 4, Null Flag = 0
      Precision = 0, Scale = 0, Collation = 1, flags_ = 0000001000001000
      Tuple Data Format = SQLARK_EXPLODED_FORMAT
      Atp = 0, AtpIndex = 1 (Temporary)
      Offset = 0, NullIndOffset = -1, VClenIndOffset = -1
      NullIndLength = 0, VClenIndLength = 0
      ValueId = 160
      Text = narrow

  PCode:

PCode Expr Length: 140
    [1]
    OPDATA_MBIN16U_IBIN32S (1) 4 0 1
    OPDATA_MPTR32_IBIN32S (0) 2 0 10
    OPDATA_MPTR32_IBIN32S (0) 4 4 11
    OPDATA_MPTR32_IBIN32S (0) 6 0 12
    CLAUSE_EVAL (108) 1448752192 32524 1
    ENCODE_MASCII_MBIN32S_IBIN32S (89) 5 0 2 0 0
    RETURN (264)

Expression: HighKeyExpr is NULL

Expression: ReturnFetchExpr is NULL

Expression: ReturnUpdateExpr is NULL

Expression: ReturnMergeInsertExpr is NULL

Expression: mergeUpdScanExpr is NULL

Expression: mergeInsertRowIdExpr is NULL

Expression: encodedKeyExpr is NULL

Expression: keyColValExpr is NULL

Expression: hbaseFilterExpr is NULL

Expression: preCondExpr is NULL

Expression: insConstraintExpr is NULL

Expression: updConstraintExpr is NULL


--- SQL operation complete.
>>